modules/instruments/README.md

```diff
@@ -40,3 +40,10 @@ instruments, but does not load through this module.
 
 `bvl_feedback` threads for an instrument may be created on this
 module.
+
+## Dashboard Widget
```

Comment:
```suggestion
### Dashboard Widget
```

The header level is wrong. The dashboard widget is an interaction of the module.